### PR TITLE
add MigrationGenerator to create moirai_translations table

### PR DESCRIPTION
**Pull request by oliveranthony17**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +33,17 @@ Mount the engine in your `config/routes.rb`:
...
 ```
 
-and run `bin/rails db:migrate`.
+Generate the necessary files including the database migration:
+
+```bash
+$ rails generator moirai:install
+```
+
+Migrate the create the table in your database:
```
Comment (coorasse) [inline README.md:42]: Strange sentence

File: README.md [hunk 2]
```diff
@@ -33,7 +33,17 @@ Mount the engine in your `config/routes.rb`:
 mount Moirai::Engine => '/moirai', as: 'moirai'
 ```
 
-and run `bin/rails db:migrate`.
+Generate the necessary files including the database migration:
```
Comment (coorasse) [inline README.md:36]: Can we also mount the engine automatically? Not for this PR. I'll add it to the questions

**Review by coorasse — APPROVED** [review]
Love it!

File: lib/moirai/engine.rb [hunk 3]
```diff
@@ -3,5 +3,9 @@
 module Moirai
   class Engine < ::Rails::Engine
     isolate_namespace Moirai
+
+    config.generators do |g|
+      g.orm :active_record
+    end
```
Comment (CuddlyBunion341) [inline lib/moirai/engine.rb:9]:
```suggestion
    config.generators do |generator|
      generator.orm :active_record
    end
```

File: README.md (hunk 1 above)
Comment (oliveranthony17) [reply README.md:42]: correct

File: README.md (hunk 2 above)
Comment (oliveranthony17) [reply README.md:36]: oooo that would be fancy! let's look in to this

**Pull request merged**